pkg/core/resources/store/options.go

```diff
@@ -57,6 +57,7 @@ func CreateWithLabels(labels map[string]string) CreateOptionsFunc {
 type UpdateOptions struct {
 	ModificationTime time.Time
 	Labels           map[string]string
+	ModifyLabels     bool
```

Comment:
Why do we need this? It's always true no?

Reply:
Seems like it's always true if `Labels != nil`?